compiler/match_/inference/type_seeder.rs

```diff
@@ -752,7 +752,7 @@ impl BinaryConstraint for Isa<Variable> {
             match left_type {
                 TypeAnnotation::Attribute(attribute) => {
                     attribute
-                        .get_supertypes(seeder.snapshot, seeder.type_manager)?
+                        .get_supertypes_transitive(seeder.snapshot, seeder.type_manager)?
```

Comment:
Thought that in
`get_supertype`
`get_supertypes` <- can look confusing 
`get_subtypes`
`get_subtypes_transitives`